vespaclient-java/src/main/java/com/yahoo/vespaget/CommandLineOptions.java

```diff
@@ -216,7 +227,7 @@ public ClientParameters parseCommandLineArguments(String[] args) throws IllegalA
                     .setTraceLevel(trace)
                     .setPriority(priority)
                     .setTimeout(timeout)
-                    .setJsonOutput(jsonOutput)
+                    .setJsonOutput(!jsonOutput && !xmlOutput ? false : jsonOutput) // TODO Vespa 7 Change default to JSON
```

Comment:
Some parenthesis-wrapping would be nice here to make operator precedence more unambiguous